### DISCUSSION TITLE
The API returns only active listings for the section

### DISCUSSION THREAD

**Discussion by nebojsapetkovic94** [etsy/open-api#1286] [Ideas]
Hi everyone,

We have been using your API call getListingsByShopSectionId, and according to the documentation, it is supposed to "Retrieve all the listings from the section of a specific shop."

However, we have noticed that it only retrieves active listings for the section. Could you please confirm if this is intentional or if it's a bug? Is there a way for us to send the status as a parameter?

Thank you.
Best regards,
Nebojsa

**Comment by Gareth064** [comment]
Unfortunately we wouldn't know if this is intentional or a bug.
Etsy Support don't monitor and answer things on here, so you'll need to email them to see if they can give you an answer.
developer@etsy.com
Let us know if they respond. Good Luck.

**Reply by Gareth064** [reply]
And as per the documentation, there is no way to pass the "State" parameter on this endpoint. If you are interested in getting inactive listings for a section then you could use the GetListingsByShop, pass in State "inactive". From the results you will see which shop section each of them is in, then grab them from there.
